examples/plotting/line_on_off.py

```diff
@@ -1,4 +1,13 @@
-""" Example demonstrating turning lines on and off - with JS only
+""" Example demonstrating turning lines on and off - with JS only.
+It involves checking and unchecking the checkboxes representing
+the plotted lines to turn the lines on/off.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.line, bokeh.layouts.row, bokeh.models.CustomJS
+    :refs: :ref:`ug_basic_lines_line_multiple`,
+    :ref:`_ug_interaction_widgets_examples_checkboxgroup`,
+    :ref:`ug_interaction_js_callbacks`
```

Comment:
Unfortunately, due to limitations in the sphinx extension, these all need to be on a single line. If the line is too long  for the linter then `# noqa` can be added to the end.